packages/site/src/index.html

```diff
@@ -7,19 +7,43 @@
     <meta name='viewport' content='width=device-width, initial-scale=1'>
     <link rel="icon" href="favicon.ico" sizes="any">
     <script src="https://unpkg.com/@ni/nimble-components@15.0.0/dist/all-components-bundle.min.js"></script>
+    <link rel="stylesheet" type="text/css" href="./styles.css" />
 </head>
 <body>
-    <div align="center">
-        <p>
-            <img width="300" src="./nimble-ui-logo.svg">
-        </p>
-        <p>
-            <nimble-breadcrumb appearance="prominent">
-                <nimble-breadcrumb-item href="./storybook">
-                    See the Storybook page
-                </nimble-breadcrumb-item>
-            </nimble-breadcrumb>
-        </p>
+    <nimble-theme-provider theme="color">
+        <div class="banner">
+            <div class="ni-logo">
+                <img src="./ni.svg" alt="NI Logo">
+            </div>
+            <div class="banner-main">
+                <img src="./nimble-logo.png" alt="Nimble Logo">
+                <p>The Nimble Design System is an easy to use collection of well-designed, styled components that can be used in any NI web application.</p>
+            </div>
+        </div>
+    </nimble-theme-provider>
+    <div class="cards-container">
+        <div class="cards">
+            <a class="card" href="https://github.com/ni/nimble">
+                <div class="icon"><img src="./gh.svg" alt="Nimble Logo"></div>
+                <div class="name">GitHub</div>
+                <div class="description">Open source access to code snippets</div>
+            </a>
+            <a class="card" href="https://nimble.ni.dev/storybook/">
+                <div class="icon"><img src="./sb.svg" alt="Nimble Logo"></div>
+                <div class="name">Storybook</div>
+                <div class="description">Search and retrieve components</div>
```

Comment:
"retrieving" components doesn't really resonate with me. Here's an alternate suggestion but feel free to leave as-is or change to something else.
```suggestion
                <div class="description">Live playground for component APIs</div>
```